dotnet/samples/Concepts/Plugins/CopilotAgentBasedPlugins.cs

```diff
@@ -8,13 +8,13 @@
 
 namespace Plugins;
 /// <summary>
-/// These examples demonstrate how to use MicrosoftManifest plugins to call Microsoft Graph and NASA APIs.
-/// Microsoft Manifest plugins are created from the OpenAPI document and the manifest file.
+/// These examples demonstrate how to use CopilotAgent plugins to call Microsoft Graph and NASA APIs.
+/// Copilot Agent Plugin plugins are created from the OpenAPI document and the manifest file.
```

Comment:
1. Should it be "Copilot Agent plugins" instead of "Copilot Agent Plugin plugins"?    
2. In line 11, "CopilotAgent" is one word, while in line 12, it's not - it's "Copilot Agent."

Reply:
fixed